docs/examples/general/data_loading/numpy_reader.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>The last paragraph sounds a bit like "if it's in fortran order, then it will be loaded in C order" - but in fact, it's always loaded in C order. There are other minor style issues, too.</p><p>Suggested rephrasing:</p><p> </p><p>The operator returns arrays with shapes taken from the files. DALI tensors are always stored in C (row-major) order. If the files contain the data in FOTRAN (column-major) order, the operator will automatically transpose the data to C order. This transposition adds significant time to the loading process. Therefore, we recommend storing files in C order when possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>